stubs/jsonschema/jsonschema/validators.pyi

```diff
@@ -86,8 +87,8 @@ class RefResolver:
     def resolution_scope(self): ...
     @property
     def base_uri(self): ...
-    def in_scope(self, scope) -> None: ...
-    def resolving(self, ref) -> None: ...
+    def in_scope(self, scope) -> AbstractContextManager[None]: ...
+    def resolving(self, ref) -> AbstractContextManager[Any]: ...
```

Comment:
- You can apparently use `@contextmanager` in stubs, just like you would use it in a `.py` file. See e.g. https://stackoverflow.com/a/49736916
- We would prefer using `Incomplete` (imported from `_typeshed`) instead of `Any` in the return type of `resolving()`. We try to use `Any` only when it isn't possible to express the correct type.